### DISCUSSION TITLE
How to use command runners.

### DISCUSSION THREAD

**Discussion by josephine-munich** [filebrowser/filebrowser#3295] [Q&A]
I'm new to this. I am trying to setup a discord webhook for file upload & download. I am not sure what commands to put for my user. I currently have a bin/bash script saved in my filebrowser folder that i'd like to use. what command should I allow my user to enter? and what do i put in the after upload / after download fields to trigger my script?

**Comment by Rapter001** [comment]
hello is it a bash sript what type of script is it and could you send that code with out your webhook url
